kubernetes/replicator-gke-cc/docs/index.rst

```diff
@@ -0,0 +1,198 @@
+-------------------
+The following applications or libraries are required to be installed and available in the system path in order to properly run the demo.
+
++------------------+----------------+----------------------------------------------------------+
+| Application      | Tested Version | Info                                                     |
++==================+================+==========================================================+
+| ``kubectl``      | ``1.14.3``     | https://kubernetes.io/docs/tasks/tools/install-kubectl/  |
++------------------+----------------+----------------------------------------------------------+
+| ``helm``         | ``2.14.3``     | https://helm.sh/docs/using_helm/#install-helm            |
++------------------+----------------+----------------------------------------------------------+
+| ``gcloud``       | ``267.0.0``    |  https://cloud.google.com/sdk/install                    |
+| ``GCP sdk core`` | ``2019.10.15`` |                                                          |
++------------------+----------------+----------------------------------------------------------+
+| ``ccloud``       | ``v0.185.0``   | https://docs.confluent.io/current/cloud/cli/install.html |
++------------------+----------------+----------------------------------------------------------+
+
+Running the Demo
+----------------
+
+.. warning:: This demo uses the real GCP SDK to launch real resources. To avoid unexpected charges, carefully evaluate the cost of resources before launching the demo and ensure all resources are destroyed after you are done evaluating the demonstration.  Optionally, refer to the |co| :ref:`Sizing Recommendations <co-env-sizing>` document and the :ref:`examples-operator-gke-base-variable-reference` section for more information on required resources for running |cp| on Kubernetes.
+ 
+Setup
+-----
+
+Clone the Confluent examples repository and change directories on your terminal into the ``replicator-gke-cc`` directory.
+
+.. sourcecode:: bash
+
+    git clone git@github.com:confluentinc/examples.git
+    cd examples/kubernetes/replicator-gke-cc
+
+GKE Setup
++++++++++
+
+In order to properly simulate a realistic replication scenario to |ccloud|, the demo requires a GKE Node Pool sufficiently large to support 3 node |zk| and 3 node |ak| clusters.  In testing this demonstration, a sufficient cluster consisted of 7 nodes of machine type ``h1-highmem-2``.  If you wish to use an existing GKE cluster, and your ``kubectl`` tool is already configured to operate with it, skip to the :ref:`quickstart-demos-operator-replicator-gke-cc-ccloud-setup` section of these instructions.
+
+The demo contains a ``make`` function to assist you in creating a cluster in GKE assuming you have your ``glcoud`` SDK properly configured to access your account.  To verify which GCP Project your ``gcloud`` SDK is currently configured to, run:
+
+.. sourcecode:: bash
+
+    gcloud config list --format 'value(core.project)'
```

Comment:
If this is repeated between gke-base and replicator-gke-cc, then make an `includes` directory and use the include directive, so it's only one copy.